index.html

```diff
@@ -20,17 +20,36 @@
   </head>
 
   <body>
-    <header>
-      <nav class="row">
-        <a href="#">agency-name-logo</a>
-        <ul class="row">
-          <a href="#about-us"><li>About us</li></a>
-          <a href="#team-members"><li>Team members</li></a>
-          <a href="#contacts"><li>Contacts</li></a>
-        </ul>
-      </nav>
-      <h1>Agency name</h1>
+    <header class="nav-bar-header">
+      <div class="nav-container">
+        <div class="logo-container">
+          <a href="index.html">
+            <img src="assets/logo-white.svg" alt="HJ logo" />

```

Comment:
- Can this SVG be embedded directly (as ``<svg>``), rather than as an ``img``?
  - (Are there pros and cons? I think an ``<svg>`` can be given a ``<title>`` element to function like alt text, for example.)
- Either way, the alt text probably needs changing?

Reply:
1.  if we put svg code from assets into html right away then, in this logo case the code in html is too long. Guess that's why put it in the img tag?
2. I put alt text for this log in footer tag `alt="creater's club white logo"`

Reply:
the svg can be embedded directly into HTML then styled in CSS. 
pros and cons depends on the use but in this case i needed it as a background img
And alt text for a true background image isn’t necessary.
In fact a true background image that adds nothing to the page content doesn’t really need to be in html as It has no ‘meaning’ that needs to be expressed for SEO  or illustration purpose. it is just visual.

this is my understanding and not sure if it's best practice.

Reply:
I hope this helps

Reply:
The SVG code is quite long, so I understand why it makes sense to not put this code directly in the HTML. Let's stick with the ``<img>`` tag for now :)

Re alt text, ``assets/logo-white.svg`` is the logo, which I think does deserve descriptive alt text. It could either be "creators club logo", or just "creators club" (which does the job of the logo being a text title, and I think might be better).


Two things to consider as nice-to-haves when we've got everything else done:

- "creators club" reads a tiny bit awkwardly, because although it's correct (we could be talking about multiple creators :), the temptation is to see a missing apostrophe, i.e. "creator's club". (Or, more confusing still, to think about the club belonging to multiple creators: "creators' club"!) I'm happy to keep the name as "creators club" but just wanted to flag this up in case anyone else wants to try variations and see what they look like in place.
- The logo text could be text (if the font is available as a web font), and only the "screwdriver"/shaker should be an image. (This probably makes it easier and faster to change the text.)